week1/index.html

```diff
@@ -0,0 +1,317 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>건휘's favorite</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable.min.css"
+    />
+</head>
+    <body>
+        <header>
+            <i class="fa-solid fa-house"></i>
+            <h1>건휘's favorite</h1>
+            <i class="fa-solid fa-bars"></i>
+        </header>
+        <nav>
+            <div class="div">
+                <h2>전체목록</h2>
+                <div class="navdiv">
```

Comment:
1. css에서 클래스 이름을 지을 때는 다 소문자로 작성하면 가독성이 떨어질 수가 있기 때문에 보통 `소문자`와 `-` 혹은 `_` 로 구분하여 사용하는 경우가 많아요 !
2. 클래스명을 지을 때는 해당 태그의 이름을 그대로 사용하는 것 보다는 어떤 부분을 나타내고 있는지를 반영해서 네이밍하는 것이 협업하기에도 좋고 나중에 본인도 코드를 보며 어느 부분이 스타일링 됐는지 알아보기에도 좋답니다! 

레이아웃의 가장 위에 순서대로
container - wrapper - item 과 같이 단계적으로 네이밍하는 습관을 들여보는 것도 좋아요 !

```suggestion
            <div class="nav_container">
                <h2>전체목록</h2>
                <div class="nav_wrapper">
```
와 같이 변경해보는 건 어떨까요 ?

Reply:
> 1. css에서 클래스 이름을 지을 때는 다 소문자로 작성하면 가독성이 떨어질 수가 있기 때문에 보통 `소문자`와 `-` 혹은 `_` 로 구분하여 사용하는 경우가 많아요 !
> 2. 클래스명을 지을 때는 해당 태그의 이름을 그대로 사용하는 것 보다는 어떤 부분을 나타내고 있는지를 반영해서 네이밍하는 것이 협업하기에도 좋고 나중에 본인도 코드를 보며 어느 부분이 스타일링 됐는지 알아보기에도 좋답니다!
> 
> 레이아웃의 가장 위에 순서대로 container - wrapper - item 과 같이 단계적으로 네이밍하는 습관을 들여보는 것도 좋아요 !
> 
> 와 같이 변경해보는 건 어떨까요 ?

예시까지 보여주면서 설명해주셔서 감사합니다! 다음부터는 클래스 이름 짓는 것도 신경써서 하겠습니다.